tests/containers/seccomp.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: seccomp
+# Summary: Test seccomp in docker & podman
+# Maintainer: QE-C team <qa-c@suse.de>
+
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use utils;
+use containers::common;
+
+sub run {
+    my ($self, $args) = @_;
+    my $runtime = $args->{runtime};
+
+    select_serial_terminal;
+    my $engine = $self->containers_factory($runtime);
+
+    assert_script_run "grep SECCOMP /boot/config-\$(uname -r)";
+    assert_script_run "$runtime info | grep -i seccomp";
+
+    my $image = "registry.opensuse.org/opensuse/bci/bci-busybox:latest";
+    my $policy = "policy.json";
+
+    # Get default policy
+    if ($runtime eq "docker") {
+        script_retry("curl -sLo $policy https://github.com/moby/moby/raw/master/profiles/seccomp/default.json", retry => 5, delay => 60, timeout => 300);
```

Comment:
For a simple file, I would put that into the `data/` directory and download it from there.

This makes the file a) static and b) avoid any network related issues at this point.

Reply:
Fixed.